Jenkinsfile

```diff
@@ -303,393 +307,61 @@ pipeline {
-        }
-        failure {
-          setGitHubBuildStatus('platform/deploy', 'Deploy Maven artifacts', 'FAILURE')
-        }
-      }
-    }
-
-    stage('Upload Nuxeo Packages') {
-      steps {
-        setGitHubBuildStatus('platform/upload/packages', 'Upload Nuxeo Packages', 'PENDING')
-        container('maven') {
-          echo """
-          ----------------------------------------
-          Upload Nuxeo Packages to ${CONNECT_PREPROD_URL}
-          ----------------------------------------"""
-          withCredentials([usernameColonPassword(credentialsId: 'connect-preprod', variable: 'CONNECT_PASS')]) {
-            sh """
-              PACKAGES_TO_UPLOAD="packages/nuxeo-*-package/target/nuxeo-*-package-*.zip"
-              for file in \$PACKAGES_TO_UPLOAD ; do
-                curl -i -u "$CONNECT_PASS" -F package=@\$(ls \$file) "$CONNECT_PREPROD_URL"/site/marketplace/upload?batch=true ;
-              done
-            """
-          }
-        }
-      }
-      post {
-        success {
-          setGitHubBuildStatus('platform/upload/packages', 'Upload Nuxeo Packages', 'SUCCESS')
-        }
-        failure {
-          setGitHubBuildStatus('platform/upload/packages', 'Upload Nuxeo Packages', 'FAILURE')
-        }
-      }
-    }
-
-    stage('Git tag and push') {
-      when {
-        not {
-          branch 'PR-*'
-        }
-      }
-      steps {
-        container('maven') {
-          echo """
-          ----------------------------------------
-          Git tag and push
-          ----------------------------------------
-          """
-          sh """
-            #!/usr/bin/env bash -xe
-            # create the Git credentials
-            jx step git credentials
-            git config credential.helper store
-
-            # Git tag
-            jx step tag -v ${VERSION}
-          """
-        }
-      }
-    }
-
-    stage('JSF pipeline') {
-      when {
-        expression {
-          // only trigger JSF pipeline if the target branch is master or a maintenance branch
-          return CHANGE_TARGET ==~ 'master|\\d+\\.\\d+'
-        }
-      }
-      steps {
-        container('maven') {
-          echo """
-          ----------------------------------------
-          Build JSF pipeline
-          ----------------------------------------
-          Parameters:
-            NUXEO_BRANCH: ${CHANGE_BRANCH}
-            NUXEO_COMMIT_SHA: ${GIT_COMMIT}
-            NUXEO_VERSION: ${VERSION}
-          """
-          build job: "/nuxeo/nuxeo-jsf-ui-status/${CHANGE_TARGET}",
-            parameters: [
-              string(name: 'NUXEO_BRANCH', value: "${CHANGE_BRANCH}"),
-              string(name: 'NUXEO_COMMIT_SHA', value: "${GIT_COMMIT}"),
-              string(name: 'NUXEO_VERSION', value: "${VERSION}"),
-            ], propagate: false, wait: false
+          setGitHubBuildStatus('nuxeo/preview/deploy', 'Deploy nuxeo Preview', 'FAILURE')
         }
       }
     }
   }
-
+  
```

Comment:
Whitespace.

Reply:
Yes, since it was in the TEMP commit I didn't mind. But it will be removed on the next push 👍